pallets/liquidity-pools-gateway-queue/src/lib.rs

```diff
@@ -93,6 +98,9 @@ pub mod pallet {
 			message: T::Message,
 			error: DispatchError,
 		},
+
+		/// Maximum number of messages was reached.
+		MaxNumberOfMessagesWasReached,
```

Comment:
Nit: Could add the current last processed nonce to the event payload for more info